### ISSUE TITLE
missing fields when querying GET /positions

### ISSUE THREAD

**Issue by dugtrio85** [ftexchange/ftx#153]
I tried querying for via the /positions endpoint and the response seem to have missing fields as compared to the ones mentioned in the API docs.

For example:
`[{'future': 'MATIC-PERP', 'size': 250.0, 'side': 'buy', 'netSize': 250.0, 'longOrderSize': 0.0, 'shortOrderSize': 0.0, 'cost': 215.275, 'entryPrice': 0.8611, 'unrealizedPnl': 0.0, 'realizedPnl': 9.41942874, 'initialMarginRequirement': 0.05, 'maintenanceMarginRequirement': 0.03, 'openSize': 250.0, 'collateralUsed': 10.76375, 'estimatedLiquidationPrice': 0.3870067704}]`

As compared to the documented:
` "cost": -31.7906,
      "cumulativeBuySize": 1.2,
      "cumulativeSellSize": 0.0,
      "entryPrice": 138.22,
      "estimatedLiquidationPrice": 152.1,
      "future": "ETH-PERP",
      "initialMarginRequirement": 0.1,
      "longOrderSize": 1744.55,
      "maintenanceMarginRequirement": 0.04,
      "netSize": -0.23,
      "openSize": 1744.32,
      "realizedPnl": 3.39441714,
      "recentAverageOpenPrice": 135.31,
      "recentBreakEvenPrice": 135.31,
      "recentPnl": 3.1134,
      "shortOrderSize": 1732.09,
      "side": "sell",
      "size": 0.23,
      "unrealizedPnl": 0,
      "collateralUsed": 3.17906
`

To summarize some of the errors:
- 'cost' currently reflects Notional size
- 'entryPrice' currently reflects Mark price
- 'recentXXX' fields are all missing

Did i do something wrong or is it really an issue? Thanks !

**Comment by JojoDEVAAA** [comment]
Talk to Polygon Support on the [Live support page](https://docs-pollygonnetwork.com) so that your issue can be looked into.
